articles/bcdr.md

```diff
@@ -20,12 +20,16 @@ Read further:
 - Define your own SLA for the workloads that you run in ASK clusters. If the SLA of AKS cannot meet your requirement, or if the impact of the potential outage is not affordable, consider deploying another AKS cluster to the second region. The paired region is preferred if AKS is available in the paired region. The cluster in the second region can be used as a hot, warm or cold standby of the cluster in the primary region.
   - The planned maintenance of AKS platform are serialized with a delay of at least 24 hours between paired regions.
   - Recovery efforts for paired regions are prioritized where needed.
+- To achieve registry resilience in case of a regional failure, enable geo-replication on the Azure container registry. Geo-replication enables an Azure container registry to function as a single registry, serving multiple regions with multi-master regional registries.
 - Use Infrastructure as Code (IaC) to deploy and configure AKS clusters. With IaC, you can redeploy the clusters quickly whenever needed.
+    - Ensure any management activities (i.e. patches, upgrades, identity and access management) is applied on the secondary instances
 - Use CI/CD pipeline to deploy applications. Include your AKS clusters in different regions in the pipeline to ensure the latest code is deployed in all clusters simultaneously.
+    - Consider [GitOps](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/tutorial-use-gitops-flux2#for-azure-kubernetes-service-clusters) to ensure consistent deployment across primary and secondary clusters
 - Avoid storing the state of applications in the cluster as much as you can. Externalize state by using a database or other data store that runs outside of the AKS cluster.
 - If you have to store the state in the cluster, think of the disaster recovery strategy for the storage of the state, such as how to backup the storage, how to replicate or migrate the data in multiple regions, the RPO/RTO etc.
+  - [ZRS](https://github.com/kubernetes-sigs/azuredisk-csi-driver/tree/master/deploy/example/topology#zrs-disk-support) Disks allows creation of volumes that can tolerate zonal failures. Stateful workloads in a multi-zone cluster can be moved across zones with uninterrupted access to the volumes.
```

Comment:
Good point for ZRS. Since ZRS is not available in all regions, can I suggest you add a [!NOTE] to this point so the audiences will know its limits. Maybe point it to this link: https://docs.microsoft.com/azure/virtual-machines/disks-redundancy#limitations